explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -289,7 +289,10 @@ object ObsTabTiles:
               PAProperties(props.obsId, guideStarSelection, agsState, props.posAngleConstraint)
 
             val averagePA: Option[AveragePABasis] =
-              (basicConfiguration.map(_.siteFor), asterismAsNel, obsDuration)
+              (basicConfiguration.map(_.siteFor),
+               asterismAsNel,
+               obsDuration.filter(_ > TimeSpan.Zero)
```

Comment:
so the calculation crashes if time is zero? Don't we have a `NonNegDuration` somewhere?

Reply:
I see